File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow.cpp

#include "SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow.h"
#include "SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow::getColumnCellLabelText( ) 
  {
    return this->columnCellLabelText;
  }
  std::string SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow::getRowHandle( ) 
  {
    return this->RowHandle;
  }
  int SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow::getRowDepth( ) 
  {
    return this->RowDepth;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow.cs
/// </filename>


namespace commands
{
    internal class SelectRowCommand_ViewMyStringRowHandleTreeViewElementsRow
  {
        private string columnCellLabelText;
     
        public     virtual string getColumnCellLabelText()
    {
      return this.columnCellLabelText;
    }
     
        private string RowHandle;
     
        public     virtual string getRowHandle()
    {
      return this.RowHandle;
    }
     
        private int RowDepth;
     
        public     virtual int getRowDepth()
    {
      return this.RowDepth;
    }
  }
}

